packages/stepper/src/EllipseStep.tsx

```diff
@@ -0,0 +1,78 @@
+import { css, cx } from '@leafygreen-ui/emotion';
+import { palette } from '@leafygreen-ui/palette';
+import Tooltip, { TriggerEvent, Align, Justify } from '@leafygreen-ui/tooltip';
+import React from 'react';
+import Step from './NewStep';
+import { StepCompletionStates, StepProps } from './types';
+
+export type EllipseStepProps = Omit<StepProps, 'state'> & {
+  startingStepIndex?: number;
+  state:
+    | StepCompletionStates.Completed
+    | StepCompletionStates.CompletedMultiple
+    | StepCompletionStates.UpcomingMultiple;
+  tooltipContent: Array<
+    React.ReactChild | React.ReactFragment | React.ReactPortal
+  >;
+};
+
+const EllipseStep = ({
+  state,
+  children,
+  tooltipContent,
+  startingStepIndex,
+  iconSize = 20,
+  ...rest
+}: React.PropsWithChildren<EllipseStepProps>) => {
+  const stepBaseStyles = css`
+    .step-icon {
+      // TODO: use centralized transition value
+      transition: 0.3s box-shadow ease;
+    }
+  `;
+
+  const completedMultipleStyles = css`
+    &:hover .step-icon {
+      // TODO: use centralized box-shadow value
+      box-shadow: 0px 0px 0px 3px ${palette.green.light2};
+    }
+  `;
+
+  const upcomingMultipleStyles = css`
+    &:hover .step-icon {
+      // TODO: use centralized box-shadow value
+      box-shadow: 0px 0px 0px 3px ${palette.gray.light2};
+    }
+  `;
+
+  const stepStyles = {
+    [StepCompletionStates.Completed]: '',
+    [StepCompletionStates.CompletedMultiple]: completedMultipleStyles,
+    [StepCompletionStates.UpcomingMultiple]: upcomingMultipleStyles,
+  };
+
+  return (
+    <Tooltip
+      align={Align.Top}
+      justify={Justify.Middle}
+      trigger={
+        <Step
```

Comment:
Currently, the `<Tooltip>` trigger doesn't work unless wrapped in a div. This is annoying because it either requires 1) the CSS rules for the connector lines to move up the component tree to the Stepper component, or 2) for those rules to be defined twice.

Going to leave the Tooltip unresolved until I can find a workaround.